psalm-baseline.xml

```diff
@@ -126,7 +124,8 @@
       <code>array_values($method)</code>
       <code>array_values($property)</code>
     </MixedArgument>
-    <MixedArgumentTypeCoercion occurrences="1">
+    <MixedArgumentTypeCoercion occurrences="2">
+      <code>$interfaces</code>
```

Comment:
The type for `$interfaces` was refined to `array<class-string>`, which obviously trips a lot of code locations where adding runtime checks now would be a major (and unnecessary) BC break.